.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+nodeLinker: node-modules
```

Comment:
If this simplifies the setup, we could also use `npm` instead of `yarn`?

Reply:
indeed, updated!